src/components/registration/join-flow-steps.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/no-multi-comp */
```

Comment:
What do you think of just naming this one of the actual steps (e.g. CountryStep or UsernameStep)?  I think it can still be a skeleton though.

Reply:
Sure, done